catalog/core/catalog-core-api/src/main/java/ddf/catalog/filter/FilterDelegate.java

```diff
@@ -1044,9 +1044,9 @@ public T propertyIsFuzzy(String propertyName, String literal) {
      * @param literal      value to compare
      * @return result of fuzzy operation
      */
-    public T propertyIs(String propertyName, Object literal) {
-        throw new UnsupportedOperationException(
-                "propertyIs(String,Object) not supported by org.opengis.filter.Filter Delegate.");
+    public T propertyIs(String propertyName, Object literal, String origMethod) {
```

Comment:
I think we should change `String origMethod` to something like `PropertyOperation type` where PropertyOperation is an enum or the strings should come from public constants so it is easy for a developer overriding this method to switch on the values.